environments/spring.py

```diff
@@ -0,0 +1,114 @@
+from matplotlib import pyplot as plt, animation
+import numpy as np
+
+from environments import Environment
+
+
+class Spring(Environment):
+
+    """Spring System
+
+    Equations of movement are:
+
+        x'' = -(k/m)*x
+
+    """
+
+    def __init__(self, mass, elastic_cst, p=None, q=None):
```

Comment:
I'd like that the order is always q, p. To try avoid integration problems with the rest of the system